css/style.css

```diff
@@ -681,3 +677,130 @@ footer .medium img {
     display: none;
   }
 }
+
+.project-pop-up {
+  padding: 5% 5%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  display: none;
+  background-image: url("/images/project-popup-bg.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.popup-container {
+  width: 100%;
+  height: auto;
+  background: rgba(255, 255, 255, 0.16);
+  border: 1px solid #fff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  padding: 24px 24px;
+}
+
+.popup-head {
+  display: flex;
```

Comment:
While you did a good job implementing the pop up window fuctionality, there are still some issues that need to be addressed. Please stick to the design on figma.

This is your design: 
![Screenshot from 2022-06-17 13-52-21](https://user-images.githubusercontent.com/79506105/174282128-9054de50-2d9c-419a-a0de-6b2deba6eae0.png)
Please kindly adjust the height of the border container of your pop up window.

Kindly Follow this design:

![Screenshot from 2022-06-17 14-34-54](https://user-images.githubusercontent.com/79506105/174282409-39e041ff-86f6-488f-8753-3ac51fb7d702.png)
Please pay attention to detail to avoid making mistakes in future

- Fix the bottoms aswel as you can see from the design the buttons are side by side not top to bottom like you have done it in you design please fix this issue thanks.